content/docs/user-guide/structure-of-remote-cache-by-example.md

```diff
@@ -0,0 +1,251 @@
+# Structure of remote cache by example
+
+This is a tutorial-like example designed to explain the remote cache structure and why it is like that. 
+If you are not familiar how DVC remote caches are structured this document will be useful to you.
+We assume you are familiar with the `git` and can use it at the command line.
+
+> While you can use any of the supported external caches, we will focus on Amazon S3 in this document.
+> You can still follow the tutorial with a different cloud service provider and a different OS, but you may need to make adjustments to the commands. 
+> It is assumed that 
+> - you have an Amazon AWS account and have set up the AWS cli on your local PC
+> - that you have set up an S3 bucket to be the remote cache
+> - you have downloaded an AWSAccessKeyId and a AWSSecretKey which you have configured on your local PC with `aws configure`.
+> - we will use a OS X/Linux CLI 
+
+
+## Set up a simple project
+
+To get started, we will set up a basic project tracked by both `git` and `dvc`. 
+The project will contain only data files and no pipelines, code or other anything else;
+the aim is to keep the project as simple as possible 
+so that the remote cache ideas can be clearly illustrated.
+
+Let's make a project called `project` and initialise it:
+
+```dvc
+$ mkdir project
+$ cd project
+$ git init
+$ dvc init
+```
+
+So far, I have not connected `git` to any external repository. 
+You can do that easily enough later on, but we won't worry about that here.
```

Comment:
Not sure what this referred to. Connect git to an external repo?

Reply:
Resolved. This was removed in a new draft.